app/experimenter/experiments/tests/test_models.py

```diff
@@ -65,6 +66,12 @@ def test_queryset_annotated_with_latest_change(self):
         )
 
 
+class TestNimbusExperimentModel(TestCase):
+    def test_str(self):
+        experiment = NimbusExperimentFactory.create(slug="experiment-slug")
+        self.assertEqual(str(experiment), experiment.name)
```

Comment:
Very nice, and I'd say we can move these tests into `experiments/models/tests/` and split them up by old and new, but we can do that as a followup as well.